src/utils/metrics.ts

```diff
@@ -46,20 +46,20 @@ export const blockHeight = new client.Gauge({
 });
 
 export const reorgDepth = new client.Gauge({
-  name: "watch_tower_reorg_depth",
-  help: "Depth of the current reorg",
+  name: "watch_tower_latest_reorg_depth",
+  help: "Depth of the last reorg",
   labelNames: ["chain_id"],
 });
 
-export const blockTime = new client.Gauge({
-  name: "watch_tower_block_time_seconds",
+export const blockProducingRate = new client.Gauge({
```

Comment:
block time i would expect the latest epoc there, with this i try to show that it tracks the producing rate